scene/gui/text_edit.cpp

```diff
@@ -3646,6 +3646,10 @@ Control::CursorShape TextEdit::get_cursor_shape(const Point2 &p_pos) const {
 			if (!is_inline_info_valid(k)) {
 				continue;
 			}
+			Dictionary info = k;
+			if (info.has("clickable") && !info["clickable"].booleanize()) {
```

Comment:
This should work too
```suggestion
			if (!info.get("clickable", false)) {
```